packages/common/src/hooks.ts

```diff
@@ -0,0 +1,35 @@
+"use client";
```

Comment:
As already described it may make sense to create a reusable function (not hook) and using https://day.js.org/docs/en/plugin/relative-time there

Reply:
Nice, I'll make the transition to Dayjs then